src/test/java/org/breedinginsight/brapps/importer/ExperimentFileImportTest.java

```diff
@@ -153,7 +157,27 @@ public void setup() {
         mappingId = (String) setupObjects.get("mappingId");
         testUser = (BiUserEntity) setupObjects.get("testUser");
         securityFp = (FannyPack) setupObjects.get("securityFp");
+        newExperimentWorkflowId = getNewExperimentWorkflowId();
+    }
+
+    /**
+     * TODO: assumes new workflow is first in list, doesn't look at position property, would be more robust to
+     * look at that instead of assuming order
+     * @return
+     */
+    public String getNewExperimentWorkflowId() {
```

Comment:
I've merged this into [BI-2134](https://github.com/Breeding-Insight/bi-api/pull/359) and renamed to ExperimentFileImportTest#getExperimentWorkflowId with an integer argument for selecting the workflow id from the list fetched by getWorkflows().